.github/workflows/developer_local.yml

```
@@ -1,17 +1,13 @@
-name: Unit test Mac
-
+name: developer at local
```

nit(feel free to ignore): 
```suggestion
name: local dev env validation
```